_source/logzio_collections/_log-sources/auditd.md

```diff
@@ -36,43 +36,46 @@ sudo curl https://raw.githubusercontent.com/logzio/public-certificates/master/Tr
 
 ##### Install Auditbeat
 
-Download and install [Auditbeat 6.8](https://www.elastic.co/guide/en/beats/auditbeat/6.8/auditbeat-installation.html).
+Download and install [Auditbeat 7.7](https://www.elastic.co/guide/en/beats/auditbeat/7.7/auditbeat-installation.html).
 
 ##### Copy auditd rules
```

Comment:
please add the following :
go to the audit rules location in you auditbeat directory
 cd /etc/auditbeat/audit.rules.d/